.github/workflows/release.yml

```
@@ -184,32 +327,19 @@ jobs:
-            - name: Create Web UI archive
-              run: |
-                  mkdir -p dist
-                  cd web
-                  tar -czf ../dist/pgedge-nla-web_${GITHUB_REF_NAME#v}_noarch.tar.gz dist/ README.md ../LICENSE.md
-                  cd ..
-                  echo "Web UI archive created:"
-                  ls -lh dist/pgedge-nla-web_*.tar.gz
-
-            - name: Generate checksums
+            - name: Assemble release assets
               run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2035:info:6:11: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)